db/access.yml

```diff
@@ -26,6 +26,7 @@ index:
 notify:
   tables:
     denylisted_notification_entities: write
```

Comment:
I think we can remove the old table name from here.

Reply:
@helfi92 Could it be needed if the db gets downgraded?

Reply:
If a downgrade is necessary, I believe we will revert the PR first before running the downgrade script. At that point, this line will have been re-added.